### PR TITLE
DOCS-1689-3.15.4

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CE 3.15.4

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 1]
```diff
@@ -1,4 +1,257 @@
 [
+  {
+    "title": "v3.15.4",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.28.12",
```
Comment (sabags) [inline calico-enterprise_versioned_docs/version-3.15/releases.json:6]:
```suggestion
      "version": "v1.28.14",
```

Comment (sabags) [reply calico-enterprise_versioned_docs/version-3.15/releases.json:6]: @danudey  does my suggestion look correct ?

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json:6]: Fixed.

Comment (sabags) [reply calico-enterprise_versioned_docs/version-3.15/releases.json:6]: I don't see the fixed commit. @bmckercher123

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json:6]: I rebased and pushed but PR doesn't seem to be updating.

**Pull request closed without merging**